src/main/java/core/basesyntax/command/Command.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.command;
+
+import core.basesyntax.ConsoleCommunicator;
+import core.basesyntax.exception.DirectoryNotExistException;
+import core.basesyntax.exception.FileNotExistException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public interface Command {
+
+    ConsoleCommunicator COMMUNICATOR = ConsoleCommunicator.getInstance();
+
+    void execute(String argument) throws IOException;
+
+    default Path getPath(String argument) {
```

Comment:
should we have this method in the command interface? I would rather add it to some FileService or smth like that

Reply:
moved to FileService